server/src/main/java/com/objectcomputing/checkins/services/checkin_notes/CheckinNoteController.java

```diff
@@ -14,92 +13,176 @@
 import io.swagger.v3.oas.annotations.tags.Tag;
 
 import javax.annotation.Nullable;
-import javax.inject.Inject;
-import javax.inject.Singleton;
 import javax.validation.Valid;
-import javax.validation.constraints.NotNull;
 import java.net.URI;
 import java.util.Set;
 import java.util.UUID;
 
+import io.micronaut.scheduling.TaskExecutors;
+import io.netty.channel.EventLoopGroup;
+import io.reactivex.Single;
+import io.reactivex.exceptions.CompositeException;
+import io.reactivex.schedulers.Schedulers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Named;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+
 
 @Controller("/services/checkin-note")
 @Secured(SecurityRule.IS_AUTHENTICATED)
 @Produces(MediaType.APPLICATION_JSON)
 @Tag(name = "checkin-note")
 public class CheckinNoteController {
+    private static final Logger LOG = LoggerFactory.getLogger(CheckinNoteController.class);
 
     private CheckinNoteServices checkinNoteServices;
+    private EventLoopGroup eventLoopGroup;
+    private ExecutorService ioExecutorService;
 
-    public CheckinNoteController(CheckinNoteServices checkinNoteServices) {
+    public CheckinNoteController(CheckinNoteServices checkinNoteServices,
+                                EventLoopGroup eventLoopGroup,
+                                @Named(TaskExecutors.IO) ExecutorService ioExecutorService) {
         this.checkinNoteServices = checkinNoteServices;
+        this.eventLoopGroup = eventLoopGroup;
+        this.ioExecutorService = ioExecutorService;
     }
 
-
     @Error(exception = CheckinNotesBadArgException.class)
     public HttpResponse<?> handleBadArgs(HttpRequest<?> request, CheckinNotesBadArgException e) {
+        LOG.info("Throwing the wrong exception");
```

Comment:
were these loggers added for testing purpose ? there are many in this class so please remove it that are unnecessary.

Reply:
Updated, removed the unnecessary logging, thanks.